Makefile

```diff
@@ -398,7 +398,7 @@ check-flake8-print:
 .PHONY: check-ruff
 check-ruff:
     # -- ruff options are determined by pyproject.toml --
-	ruff .
+	ruff check .
```

Comment:
This fixes a deprecation warning in CI. `ruff` now also does reformatting, like Black, so there is `ruff check ...` and `ruff format ...`. We only want the lint part (check) for present purposes.